### PR TITLE
Fixes issues with imports

### PR DESCRIPTION
**Pull request by arlyon**

### CODE REVIEW

File: starling_server/main.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 db = Database()
 controller = Controller(db=db)
 
+def run():
+    uvicorn.run("starling_server.server.app:app", host="0.0.0.0", port=8000, reload=True)
+
+
 if __name__ == "__main__":
-    uvicorn.run("server.app:app", host="0.0.0.0", port=8000, reload=True)
+    run()
```
Comment (richardjlyon) [inline starling_server/main.py]: Add a newline at the end of each file.

File: tests/db/test_edgedb.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def test_insert_or_update_account_insert_2(self, db):
         # THEN two accounts are added
         accounts = make_accounts(2)
         for account in accounts:
-            db.insert_or_update_account(account)
+            starling_server.db.insert_or_update_account(account)
```
Comment (richardjlyon) [inline tests/db/test_edgedb.py]: This looks like a mistake - should be db.etc

**Review by richardjlyon — CHANGES_REQUESTED** [review]
Thank Alex - made a couple of suggestions - please review and implement.

File: tests/db/test_edgedb.py (hunk 2 above)
Comment (arlyon) [reply tests/db/test_edgedb.py]: Ah yes, you're correct, I'll fix that now.

**Review by richardjlyon — APPROVED** [review]
Good job!

**Pull request merged**